selfdrive/car/interfaces.py

```diff
@@ -33,6 +34,16 @@ def update(self, c, can_strings):
   def apply(self, c):
     raise NotImplementedError
 
+  def create_common_events(self, c, cs_out):
```

Comment:
really like this abstraction. A couple of things:
- since does not depend on self, a staticmethod could be more appropriate.
- I think a lot more standard events can be abstracted: espDisable, wrongMode, wrongGear, reverseGear. 
- With some more work, some other events can be commonized too (I can do this). 

what do you think?

Reply:
That's why this was supposed to be a draft :)

There's definitely more that can be abstracted here. I went back and did a few quick ones. The other events have a couple minor differences between the interfaces